### PR TITLE
Adds caching to the FITS role

### PR DESCRIPTION
**Pull request by rotated8**

### CODE REVIEW

File: ansible/roles/fits/tasks/main.yml [hunk 1]
```diff
@@ -7,6 +7,13 @@
   with_items:
     - unzip
 
+- name: copy cached fits
+  copy:
+    src: '{{ local_files_dir }}/fits-{{ fits_version }}.zip'
+    dest: /tmp/fits-{{ fits_version }}
```
Comment (pmather) [inline ansible/roles/fits/tasks/main.yml]: Is this missing a ".zip" on the end?

**Review by pmather — COMMENTED** [review]
I think this line is missing a ".zip" on the end of the destination.  Elsewhere in the tasks it is referred to as `/tmp/fits-{{ fits_version }}.zip`.

File: ansible/roles/fits/tasks/main.yml (hunk 1 above)
Comment (rotated8) [reply ansible/roles/fits/tasks/main.yml]: Good catch!

**Comment by rotated8** [comment]
@pmather Updated.

**Pull request merged**